docs/development/setup_development.rst

```diff
@@ -3,7 +3,7 @@ Setting Up the Development Environment
 
 .. include:: ../includes/docs-branches.txt
 
-Prerequisites
+Overview
 -------------
```

Comment:
Nit: It makes no difference in the output, but for readability, I would recommend shortening the length of the underline (the `-------------`) to match the length of the headline.